crates/uv-cli/src/lib.rs

```diff
@@ -1940,36 +1953,43 @@ impl ExternalCommand {
 #[allow(clippy::struct_excessive_bools)]
 pub struct InitArgs {
     /// The path of the project.
+    ///
+    /// Relative and absolute paths are allowed.
```

Comment:
I might say "Accepts both relative and absolute paths." Something something passive voice.